### ISSUE TITLE
TensorFlow2IR 

### ISSUE THREAD

**Issue by zhurb88** [Microsoft/MMdnn#206]
Platform (CentOS 6.6):

Python version:2.7.14

Source framework with version (like Tensorflow 1.6.1 with GPU):

Pre-trained model path (own):

Running scripts:
python -m mmdnn.conversion._script.convertToIR -f tensorflow -d resnet101 -n tests/resnet_v1_101.ckpt.meta --dstNodeName muli_predictions
![image](https://user-images.githubusercontent.com/13047093/40397788-306ffe34-5e67-11e8-82a8-e57bc5661d8c.png)
The last second lay of the net must be "dropout"?

**Comment by kitstar** [comment]
Hi @zhurb88, In our test set tensorflow models, the RealDiv is only used in dropout layers. It is easy to implement the parser for other cases but we don't have model to test the correctness. Could you provide some?

**Comment by zhurb88** [comment]
It‘s a opensource. https://storage.googleapis.com/openimages/2017_07/oidv2-resnet_v1_101.ckpt.tar.gz.

**Comment by JiahaoYao** [comment]
@zhurb88 
I used `mmvismeta` (if not familiar, one can refer to [Tensorflow Readme](https://github.com/Microsoft/MMdnn/blob/master/mmdnn/conversion/tensorflow/README.md) ) to visualize the model. 
![image](https://user-images.githubusercontent.com/20907377/40537742-40abbe00-6042-11e8-8363-0e0830e10fb1.png)

The point is that the model contains preprocessing of image inside the check point. However, MMdnn only converts the **neural net part** without loss of generality. I think the preprocessing might be similar to [Open image classify file](https://github.com/openimages/dataset/blob/master/tools/classify.py#L57-L80). You may take it as a reference if converted to other framework.

----------

Here is one practical way to convert the main body of the model to IR.

I truncate the original graph into the subgraph. Choosing the `map/TensorArrayStack/TensorArrayGatherV3` as the input, and `resnet_v1_101/logits/BiasAdd` as the output.
